timemachine/fe/plots.py

```diff
@@ -322,3 +322,107 @@ def plot_fwd_reverse_predictions(
     plot_png = buffer.read()
 
     return plot_png
+
+
+def plot_hrex_transition_matrix(transition_rate: NDArray):
```

Comment:
Consistency question: Should these return bytes like the other plotting functions in this file? No opinion really, but would be good to be consistent

Reply:
It seems like we have a mix of existing conventions in this file. Personally I think I'd prefer to decouple the drawing of the plots with serialization to bytes, since the former would be nicer to use in notebooks and scripts.